pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/OneOf.java

```diff
@@ -49,5 +49,13 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(kind, value);
     }
+
+    @Override
+    public int compareTo(OneOf<E> thatObj) {
+        if(thatObj == null) {
+            return 1;
+        }
+        return kind.compareTo(thatObj.kind);
```

Comment:
If kinds are equal, shouldn't values be compared next?

Reply:
The values would need to be compared in order to be correct.
If that's too much for this first-pass, we should reject any message with OneOf structures until that's supported (in a future PR, most likely).

Reply:
`AccountID` needs to be comparable and it contains a `OneOf` structure.

Reply:
I agree.  I also know this is quite a complex PR. If we can get a correct, but limited, result quickly, then add the OneOf and bytes handling in a follow-up; I believe that may be a good path forward.

Not my call, but perhaps worth considering.

Reply:
Fixed